compiler/crates/relay-lsp/src/completion/mod.rs

```diff
@@ -710,6 +753,7 @@ fn resolve_completion_items_for_inline_fragment_type(
     type_: Type,
     schema: &SDLSchema,
     existing_inline_fragment: bool,
+    include_fragment: bool,
```

Comment:
nit: boolean arguments maybe confusing sometimes.

Maybe we should change this into the callback -> `format_fragment_completion_label`?

Reply:
I completely agree about boolean arguments, I rewrote with a callback and it seemed more confusing

We already use a boolean argument in a similar situation for `resolve_completion_items_for_fragment_spread`. Here `existing_fragment_spread: bool` is used to describe whether the completion is being used to rename an existing spread.

I've renamed this function and argument for inline fragments to more closely align to the existing one for fragment spreads. Now the argument is `existing_fragment_spread`.